src/Symfony/Bundle/FrameworkBundle/Test/KernelTestCase.php

```diff
@@ -106,6 +106,14 @@ private static function getPhpUnitCliConfigArgument()
      */
     protected static function getKernelClass()
     {
+        if (isset($_SERVER['KERNEL_CLASS'])) {
+            if (!class_exists($class = $_SERVER['KERNEL_CLASS'])) {
+                throw new \RuntimeException(sprintf('Class "%s" not found or cannot be autoloaded. Please check the KERNEL_CLASS value in your phpunit.xml matches the fully-qualified class name of your Kernel or override the WebTestCase::createKernel() method.', $class));
```

Comment:
Updated. Thank you :)

I also updated the `WebTestCase::createKernel()` part to use `static::class` instead of `WebTestCase`, so the message will indicate the final class. Anyway it was wrong, as we're in `KernelTestCase`, so I might propose a fix on lower branches too for the other exception a few lines below.